package.json

```diff
@@ -184,6 +185,7 @@
     "jsonpath": "^1.0.2",
     "lodash": "^4.17.15",
     "mac-ca": "^1.0.4",
+    "make-synchronous": "^0.1.0",
```

Comment:
Why do we need this?

Reply:
Because `file-type` is async only but migrations have to be sync only